hygen/_templates/layer/new/prompt.js

```diff
@@ -56,7 +56,7 @@ const prompt = async ({ prompter, args }) => {
 
   // Detect what kind of layer we need (CartoSQLLayer, CartoBQTilerLayer)
   const selectedSourceFileContent = readFile(`src/data/sources/${answers.source_file}.js`);
-  const res = /(?:type: ')(?<type>sql|bq*)(?:')/g.exec(selectedSourceFileContent);
+  const res = /(?:type: ')(?<type>sql|bigquery*)(?:')/g.exec(selectedSourceFileContent);
```

Comment:
If you make this dynamic, using the lib enum, you can use something like https://stackoverflow.com/questions/5090103/javascript-regexp-dynamic-generation-from-variables 

And please double check the resulting regex, we already had an issue with this line

Reply:
As discussed, cannot require npm packages here, will wait for the creation of the lib cdn.

Reply:
I think we could just require from local node_modules but, in any case, there are some other hardcoded uses in _templates/sources/new, that assume a certain order and have more hardcoded metadata, so it's not feasible now.

Reply:
...not so direct, so we'll skip this

Reply:
Seems a bit weird requiring a relative path from node_modules..